app/javascript/packages/document-capture/components/acuant-selfie-capture-canvas.jsx

```diff
@@ -17,13 +17,20 @@ function FullScreenLoadingSpinner({ fullScreenRef, onRequestClose, fullScreenLab
   );
 }
 
-function AcuantSelfieCaptureCanvas({ fullScreenRef, onRequestClose, fullScreenLabel }) {
+function AcuantSelfieCaptureCanvas({
+  fullScreenRef,
+  onRequestClose,
+  fullScreenLabel,
+  imageCaptureText,
+}) {
   const { isReady } = useContext(AcuantContext);
   // The Acuant SDK script AcuantPassiveLiveness attaches to whatever element has
   // this id. It then uses that element as the root for the full screen selfie capture
   const acuantCaptureContainerId = 'acuant-face-capture-container';
   return isReady ? (
-    <div id={acuantCaptureContainerId} />
+    <div id={acuantCaptureContainerId}>
+      <p className="document-capture-selfie-feedback">{imageCaptureText}</p>
```

Comment:
Our use of [BEM naming conventions](https://github.com/18F/identity-idp/blob/main/docs/frontend.md#naming) aligns neatly 1-to-1 with React components, where if we had a CSS class name of `document-capture-selfie-feedback`, I'd expect it to be assigned in a component `DocumentCaptureSelfieFeedback`. I think that could make sense to split out as a component. As currently implemented, I'd assume it would be called something like `acuant-selfie-capture-canvas__selfie-feedback`, being an element (B*E*M) within `AcuantSelfieCaptureCanvas`.